### DISCUSSION TITLE
Would it be possible to remove my bridged account on bluesky. Im from nostr

### DISCUSSION THREAD

**Discussion by issanode** [community/community#129889] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

My nostr npub is : npub1rzg96zjavgatsx5ch2vvtq4atatly5rvdwqgjp0utxw45zeznvyqfdkxve

thank you for your help

**Comment by ebndev** [comment]
Hi @issanode, thanks for being a part of the GitHub Community!

If you're looking for help with this, you might want to try contacting their support instead of GitHub.

We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
